docs/reference/analysis/tokenfilters/synonym-graph-tokenfilter.asciidoc

```diff
@@ -37,7 +37,8 @@ PUT /test_index
                 "filter" : {
                     "graph_synonyms" : {
                         "type" : "synonym_graph",
-                        "synonyms_path" : "analysis/synonym.txt"
+                        "synonyms_path" : "analysis/synonym.txt",
+                        "tokenizer": "whitespace"
```

Comment:
Could you please add a callout <1> to the example that you added?

Example is https://raw.githubusercontent.com/elastic/elasticsearch/master/docs/reference/analysis/analyzers/configuring.asciidoc